zsl/interface/webservice/utils/error_handler.py

```diff
@@ -42,15 +45,16 @@ def error_handling_function(*args, **kwargs):
         try:
             return f(*args, **kwargs)
         except ImportError as ie:
-            service_application.logger.error(str(ie) + "\n" + traceback.format_exc())
+            logging.error(str(ie) + "\n" + traceback.format_exc())
             return str(ie), 404
```

Comment:
This should be outputted only in debug mode. Otherwise we should output something less informative.